tx_submitoor/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"flag"
+
+	eigenpodproofs "github.com/Layr-Labs/eigenpod-proofs-generation"
+	txsubmitter "github.com/Layr-Labs/eigenpod-proofs-generation/tx_submitoor/tx_submitter"
+	"github.com/caarlos0/env"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+type Config struct {
+	RPC          string `json:"RPC_URL"`
+	PrivateKey   string `json:"PRIVATE_KEY"`
+	ChainID      uint64 `json:"CHAIN_ID,required"`
+	CacheExpire  int    `json:"CACHE_EXPIRE" envDefault:"1000"`
+	BeaconAPIURL string `json:"BEACON_API_URL"`
```

Comment:
Hi, with how your parseConfig() function is written, I believe you want to use the struct tag env and not json here